resources/codedeploy/scripts/install.sh

```diff
@@ -6,6 +6,6 @@ do
   sleep 2
 done
 
-# aws s3 cp s3://ec-europa-db-dump/joinup/joinup_prod_cleaned.sql ./tmp/d6-joinup.sql
+aws s3 cp s3://ec-europa-db-dump/joinup/joinup_prod_cleaned.sql ./tmp/d6-joinup.sql --region=eu-west-1
 vendor/bin/phing -propertyfile /usr/local/etc/subsite/subsite.ini install setup-acceptance  >> /var/log/subsite-install.log 2>&1
```

Comment:
I think this should run under www-data. Not sure but I remember having issues.

{code}
sudo -u www-data vendor/bin/phing -propertyfile /usr/local/etc/subsite/subsite.ini install setup-acceptance  >> /var/log/subsite-install.log 2>&1
{code}